vm/systemSmartContracts/staking.go

```diff
@@ -297,16 +308,20 @@ func (r *stakingSC) changeRewardAddress(args *vmcommon.ContractCallInput) vmcomm
 
 func (r *stakingSC) unJail(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 	if !bytes.Equal(args.CallerAddr, r.stakeAccessAddr) {
-		log.Debug("stake function not allowed to be called by", "address", args.CallerAddr)
+		log.Debug("unJail function not allowed to be called by", "address", args.CallerAddr)
+
+		r.eei.SetReturnMessage("unJail function not allowed to be called by" + "address " + string(args.CallerAddr))
```

Comment:
add white space between by and address, remove concatenation

Reply:
done.